### PR TITLE
Finalize development of About us section

### PR DESCRIPTION
**Pull request by sebastian-ew**
## Description

This PR includes a custom menu template based on the parent theme menu template. Also, includes styles for the different pages of the About us section.

## Related ticket

https://3.basecamp.com/3451303/buckets/37454340/todos/7379342286

> Make sure to read the *Pull requests process* in CONTRIBUTING.md.
> 
> Main points:
> 
> * The assigned reviewer(s) must always *submit* a review (not simply add a
>   comment).
>
> * If the PR is "Approved", the creator of the PR must then:
>   * Implement the changes in the reviewer's comments, if applicable, and mark
>     them as "Resolved".
>   * Squash and merge the branch.
>   * Delete the branch.
> 
> * If the PR is "Request Changes", the creator of the PR must then:
>   * Update the code as requested.
>   * Re-request a review from the reviewer.
>   * NOT mark the comments as "Resolved", the reviewer will.
> 
> (You can keep or remove this block, as you wish.)

### CODE REVIEW

**Review by MahieddineAk — COMMENTED** [review]
This is looking good to me.

**Review by xfrenette-crim — CHANGES_REQUESTED** [review]
Small changes requested.

File: fw-child/template/aside-collapsible-menu.php [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        );
+    }
+
+    $menu = fw_build_menu ( $menu_items, $build_menu_top, 1 );
+
+}
+
+?>
+
+<div class="climatedata-menu">
```
[the remark this answers is not in the document]
Comment (sebastian-ew) [reply fw-child/template/aside-collapsible-menu.php]: @xfrenette-crim this class name was changed. Can you take a look again? Thanks!

File: fw-child/template/aside-collapsible-menu.php [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+	<?php
+        function climatedata_aside_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
```
[the remark this answers is not in the document]
Comment (sebastian-ew) [reply fw-child/template/aside-collapsible-menu.php]: @xfrenette-crim prefix was changed following the recommendation.

[the remark this answers is not in the document]
Comment (sebastian-ew) [reply fw-child/template/aside-collapsible-menu.php]: @xfrenette-crim a comment was added with a short description and the parameters.

File: fw-child/resources/scss/_template.scss [hunk 3]
```diff
@@ -287,3 +287,70 @@
     display: none;
   }
 }
+
+//
+// ASIDE MENU
+//
+
+.climatedata-menu {
+  .climatedata-collapsible-item {
```
[the remark this answers is not in the document]
Comment (sebastian-ew) [reply fw-child/resources/scss/_template.scss]: @xfrenette-crim the sub classes are not prefixed anymore. Can you check again? Thanks!

File: fw-child/template/aside-collapsible-menu.php
[the remark this answers is not in the document]
Comment (sebastian-ew) [reply fw-child/template/aside-collapsible-menu.php:1]: @xfrenette-crim the file was renamed following your suggestion.

File: fw-child/template/hierarchical-collapsible-menu.php [hunk 4]
```diff
@@ -40,20 +40,32 @@
...
+         * @param array $menu_structure The entire menu structure generated
+         * @param boolean $is_submenu Defines whether the current tree is a
+         *                            submenu or not
+         * @param int $submenu_id Optional. Ordinal to create unique ID for
+         *                        submenu. Default: null
+         * @param boolean $in_active_trail Optional. Defines if the current
+         *                                 menu is part of the active trail.
+         *                                 Default: null
+         */
+        function cdc_aside_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
```
Comment (xfrenette-crim) [inline fw-child/template/hierarchical-collapsible-menu.php:57]: Maybe use "hierarchical" in the function's name:
```suggestion
        function cdc_hierarchical_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
```

File: fw-child/template/hierarchical-collapsible-menu.php [hunk 5]
```diff
@@ -40,20 +40,32 @@
...
 	<?php
-        function climatedata_aside_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
+        /**
+         * Displays the menu hierarchy and makes it collapsible.
+         *
+         * @param array $menu_structure The entire menu structure generated
+         * @param boolean $is_submenu Defines whether the current tree is a
+         *                            submenu or not
+         * @param int $submenu_id Optional. Ordinal to create unique ID for
+         *                        submenu. Default: null
```
Comment (xfrenette-crim) [inline fw-child/template/hierarchical-collapsible-menu.php:52]:
```suggestion
         *                        submenu. Default: NULL
```

File: fw-child/template/hierarchical-collapsible-menu.php [hunk 6]
```diff
@@ -40,20 +40,32 @@
...
+         * @param boolean $in_active_trail Optional. Defines if the current
+         *                                 menu is part of the active trail.
+         *                                 Default: null
```
Comment (xfrenette-crim) [inline fw-child/template/hierarchical-collapsible-menu.php:55]:
```suggestion
         *                                 Default: NULL
```

File: fw-child/template/hierarchical-collapsible-menu.php [hunk 7]
```diff
@@ -40,20 +40,32 @@
...
-<div class="climatedata-menu">
+<div class="hierarchical-menu">
 	<?php
-        function climatedata_aside_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
+        /**
+         * Displays the menu hierarchy and makes it collapsible.
+         *
+         * @param array $menu_structure The entire menu structure generated
+         * @param boolean $is_submenu Defines whether the current tree is a
+         *                            submenu or not
```
Comment (xfrenette-crim) [inline fw-child/template/hierarchical-collapsible-menu.php:50]:
```suggestion
         *                            submenu or not.
```

File: fw-child/template/hierarchical-collapsible-menu.php [hunk 8]
```diff
@@ -40,20 +40,32 @@
...
 ?>
 
-<div class="climatedata-menu">
+<div class="hierarchical-menu">
 	<?php
-        function climatedata_aside_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
+        /**
+         * Displays the menu hierarchy and makes it collapsible.
+         *
+         * @param array $menu_structure The entire menu structure generated
```
Comment (xfrenette-crim) [inline fw-child/template/hierarchical-collapsible-menu.php:48]:
```suggestion
         * @param array $menu_structure The entire menu structure generated.
```

File: fw-child/template/hierarchical-collapsible-menu.php [hunk 9]
```diff
@@ -143,7 +155,7 @@ function climatedata_aside_menu( $menu_structure, $is_submenu, $submenu_id = NUL
             echo '</ul>';
         }
 
-        climatedata_aside_menu( $menu, false );
+        cdc_aside_menu( $menu, false );
```
Comment (xfrenette-crim) [inline fw-child/template/hierarchical-collapsible-menu.php:158]:
```suggestion
        cdc_hierarchical_menu( $menu, false );
```

File: fw-child/resources/scss/_template.scss [hunk 10]
```diff
@@ -287,3 +287,70 @@
...
+  .button-expand {
+    background-color: transparent;
+    border: none;
+    color: $secondary;
+    display: inline-block;
+    padding: .375rem .5rem;
+    transition: background-color .2s;
+
+    &:hover {
+      background-color: $gray-400;
```
Comment (xfrenette-crim) [inline fw-child/resources/scss/_template.scss:312]: (Sorry, a comment I forgot to add at last review). The hover effect on the menu (defined here and in the `a` below) is not found anywhere else. It feels like a different design just for this menu.

I suggest to leave the default style for the link colors and the hover effects. When we will do the QA on staging, we can reevaluate if we want to change the style.

**Review by xfrenette-crim — APPROVED** [review]
Approved, but see comments.

File: fw-child/template/hierarchical-collapsible-menu.php (hunk 4 above)
Comment (sebastian-ew) [reply fw-child/template/hierarchical-collapsible-menu.php]: @xfrenette-crim changed here and in the other places where it's needed.

File: fw-child/template/hierarchical-collapsible-menu.php (hunk 5 above)
Comment (sebastian-ew) [reply fw-child/template/hierarchical-collapsible-menu.php]: @xfrenette-crim fixed. Thanks!

File: fw-child/template/hierarchical-collapsible-menu.php (hunk 6 above)
Comment (sebastian-ew) [reply fw-child/template/hierarchical-collapsible-menu.php]: @xfrenette-crim also fixed. Thanks!

File: fw-child/template/hierarchical-collapsible-menu.php (hunk 7 above)
Comment (sebastian-ew) [reply fw-child/template/hierarchical-collapsible-menu.php]: @xfrenette-crim changed too. Thanks!

File: fw-child/template/hierarchical-collapsible-menu.php (hunk 8 above)
Comment (sebastian-ew) [reply fw-child/template/hierarchical-collapsible-menu.php]: @xfrenette-crim changed. Thanks!

File: fw-child/template/hierarchical-collapsible-menu.php (hunk 9 above)
Comment (sebastian-ew) [reply fw-child/template/hierarchical-collapsible-menu.php]: @xfrenette-crim changed too. Thanks!

File: fw-child/resources/scss/_template.scss (hunk 10 above)
Comment (sebastian-ew) [reply fw-child/resources/scss/_template.scss:312]: @xfrenette-crim I removed the custom styles for the links and left it as the default. My only concern on this matter is how to make more noticeable the active link in the menu. I changed the active link to use the hover state of the link, but it's not easy to differentiate from the regular links.

Comment (xfrenette-crim) [reply fw-child/resources/scss/_template.scss]: @domcode-ew I will add this point in the Basecamp discussion. I also pushed a style update for the hover effect of the "collapsible" button.

**Review by xfrenette-crim — APPROVED** [review]

**Pull request merged**